net/upnp/src/etc/inc/plugins.inc.d/miniupnpd.inc

```diff
@@ -150,7 +150,11 @@ function miniupnpd_configure_do($verbose = false)
             if ($if != $iface) {
                 $addr = get_interface_ip($iface);
                 if (!empty($addr)) {
-                    $config_text .= "listening_ip={$if}\n";
+                    if(isset($upnp_config['overridesubnet'])) {
+                        $config_text .= "listening_ip={$if}{$upnp_config['overridesubnet']}\n";
```

Comment:
what sort of data is implied here? there's no separation between first and second variable

Reply:
Didn't mean to start a formal review, leaving comment here to explain this one.

This is to allow the user to override the subnet as defined by opnsense so that miniupnpd can respond to upnp requests. Useful if running miniupnpd across VLANs for example.

see: https://github.com/miniupnp/miniupnp/issues/330#issuecomment-434781573